util/log/clog.go

```diff
@@ -844,7 +844,7 @@ func (l *loggingT) getTermColorProfile() *colorProfile {
 			case "ansi", "xterm-color":
 				l.colorProfile = colorProfile8
 				color = true
-			case "xterm-256color":
+			case "xterm-256color", "screen-256color":
```

Comment:
I have no idea honestly, I thought everything used "xterm" at this point, but apparently tmux and screen do not.  The entire thing is confusing.